packages/zoe/src/zoe.js

```diff
@@ -147,36 +168,60 @@ import { makeTables } from './state';
  * @typedef {Object} ContractInstance
  * @property {Invite} invite The closely-held administrative invite
  * @property {Object.<string,function>} publicAPI Public functions that can be called on the instance
+ */
+
+/**
+ * @typedef {{}} InstanceHandle - an opaque handle for a contract instance
+ * @typedef {{}} OfferHandle - an opaque handle for an offer
+ * @typedef {{}} InviteHandle - an opaque handle for an invite
+ * @typedef {Object} CustomProperties
  *
- * @typedef {Object} InstanceHandle
- * @typedef {Object} OfferHandle
- * @typedef {Object} InviteHandle
+ * @typedef {object} OfferRecord
+ * @property {OfferHandle} handle - opaque identifier for the offer, used as the key
+ * @property {InstanceHandle} instanceHandle - opaque identifier for the instance
+ * @property {Proposal} proposal - the offer proposal (including want, give, exit)
+ * @property {AmountKeywordRecord} amounts - the amountKeywordRecord that will be turned into payouts
```

Comment:
```suggestion
 * @property {AmountKeywordRecord} currentAllocation - the amountKeywordRecord that will be turned into payouts
```

Reply:
Should we add notifier and updater here?

Reply:
Ok, that makes sense to me. Since I had two comments here, I just want to make sure the first suggested change comment doesn't get lost.

Reply:
I like the format of the built-in suggestions, but I haven't figured out how to acknowledge them when I'm making the change with a bunch of others on my side rather than hitting the github buttons.

I changed `amounts` to `currentAllocation`.